uktrade/great-cms
# KLS-741 - Display summary of filters applied on event listing page

This PR adds a 'results summary' at the top of the event listing page. This summary changes depending on whether the user is looking at all events, current bookings or past bookings.

**To test locally**
- Go to `export-academy/events`
- Apply the filters to see the filter summary appear
- If you don't have any events appearing - make sure to publish some future events locally [here](http://greatcms.trade.great:8020/admin/events/)

### **All events**
![Screenshot 2023-06-12 at 15 45 52](https://github.com/uktrade/great-cms/assets/22460823/aa112f38-115a-4b74-8ff0-c7ea21ae8fd0)

**All events no results**
![Screenshot 2023-06-12 at 15 45 17](https://github.com/uktrade/great-cms/assets/22460823/60b12987-2702-4aa2-bb23-08f41873377c)

### **Current bookings**
![Screenshot 2023-06-12 at 15 46 57](https://github.com/uktrade/great-cms/assets/22460823/afa307de-c060-4d36-9b3b-c20d727fd7f5)
**Current bookings no results**
![Screenshot 2023-06-12 at 15 47 12](https://github.com/uktrade/great-cms/assets/22460823/9ead798f-7fe1-4159-83cc-be5c18b7fbdd)

### **Past bookings**
![Screenshot 2023-06-12 at 15 48 55](https://github.com/uktrade/great-cms/assets/22460823/02d09565-9ac0-4ac8-a19b-5d90de409837)

![Screenshot 2023-06-12 at 15 48 38](https://github.com/uktrade/great-cms/assets/22460823/5250dda3-bb39-4897-a7b8-71fa6dd8f829)



### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/KLS-741

### Reviewing help


### Housekeeping


### Merging
